web/client/reducers/__tests__/draw-test.js

```diff
@@ -8,6 +8,7 @@
 import expect from 'expect';
 
 import draw, {defaultSnappingConfig} from '../draw';
+import { normalizeGeomerty } from '../../utils/ImmutableUtils';
```

Comment:
please remove this
it was good as you did before

if you apply changes to the reducers you will notice 
that you will have to fix tests were you do the check
expect(state.tempFeatures[0]).toEqual(feature1);